collection-scripts/targeted_crs

```diff
@@ -119,12 +119,21 @@ if [ ! -z "${vm_resources}" ]; then
 
         # Gather VMs PersistentVolumeClaims
         for pvc_name in $(echo $vm_data | jq -r '.spec.template.spec.volumes[] .persistentVolumeClaim.claimName'); do
-          # In order to gather the scratch PVCs in warm migrations
-          for pvc in $(/usr/bin/oc get pvc -n ${target_ns} -o json | jq -c '.items | .[]'); do
-            if [ $(echo $pvc | jq -r '.spec.volumeName') != $pvc_name ]; then
+          pvcs=$(/usr/bin/oc get pvc -n ${target_ns} -o json | jq -c '.items | .[]')
+          pop_volume_name=""
+          for pvc in ${pvcs[@]}; do
+            # Volume populator prime PVC
+            if [ $(echo $pvc | jq -r '.metadata.name') = "$pvc_name" ]; then
+              pop_volume_name=$(echo $pvc | jq -r '.spec.volumeName')
+            fi
+          done
+          for pvc in ${pvcs[@]}; do
+            # In order to gather the scratch PVCs in warm migrations
```

Comment:
and here we should say that this holds for the `[ "$volume_name" != "$pvc_name" ]` part (maybe to extract it to a separate line)

Reply:
i made a clearer comment. please check it.